packages/react-ui/src/widgets/FeatureSelectionWidgetUI/FeatureSelectionWidgetUI.js

```diff
@@ -47,7 +47,7 @@ function FeatureSelectionWidgetUI(props) {
     editModes = [],
```

Comment:
Shouldn't this be `EMPTY_ARRAY` as per above changes?

Reply:
Ah good catch, thank you!